sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/models/TrainingDocumentInfo.java

```diff
@@ -14,6 +14,7 @@
  */
 @Immutable
 public final class TrainingDocumentInfo {
+    private String modelId;
```

Comment:
a new constructor needed to set `modelId`

Reply:
Same as this https://github.com/mssfang/azure-sdk-for-java/pull/41#discussion_r490439055